CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [1.4.1] - TBC
+### Fixed
+* Made newTablePathResolver method in TablePathResolver public.
```

Comment:
We generally put method and classnames in `` (`newTablePathResolver` and `TablePathResolver`) to improve readability, I'm approving PR as is but would be nice if you made that small change.